scripts/forger/seed/download_seed.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+# This script is used to download and extract the blockchain data from a seed file.
+set -eEuo pipefail
+
+command -v aria2c &>/dev/null || {
+  echo "${FUNCNAME[0]} Error: 'aria2c' is required to run this script, see installation instructions at 'https://github.com/aria2/aria2/releases/tag/release-1.37.0'."
+  exit 1
+}
+
+script_dir="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+deployment_dir="$(readlink -f "${script_dir}/../")"
+seed_dir="${deployment_dir}/seed"
+env_file="${deployment_dir}/.env"
+
+# shellcheck source=../.env
+source "${env_file}"
+
+remote_file="${ZEN_SEED_TAR_GZ_URL}"
+filename=$(basename "${remote_file}")
+
+content_length=$(curl -s -I -X HEAD -H "Range: bytes=0-1023" "${remote_file}" | grep content-length | awk '{print $2}' | tr -d '\r')
+extended_content_length=$((content_length + content_length / 10))
+total_size=$((2 * extended_content_length))
+
+{
+  echo "Seed file size: ${content_length} bytes"
+  echo "Minimal required storage size: ${extended_content_length} bytes"
+  echo "Recommended storage size: ${total_size} bytes"
+} >>"${seed_dir}/seed.log"
```

Comment:
Non-blocking, directing to both stdout and the logfile would provide more information on progress to the user.

`| tee -a "${seed_dir}/seed.log"` instead of `>>"${seed_dir}/seed.log"` everywhere we're redirecting.

Reply:
Done.